harness/determined/__init__.py

```diff
@@ -17,6 +17,7 @@
     InvalidHP,
 )
 from determined import errors
+from determined import pytorch
```

Comment:
no adding mandatory frameworks to determined.  This would be very annoying to pure tensorflow users, or pure core api users who might be using something like scikit learn and neither pytorch nor tensorflow.